features/foutafhandeling.feature

```diff
@@ -0,0 +1,135 @@
+        | code   | paramsValidation                              |
+        | detail | <detail>                                      |
+        En bevat de response de volgende invalidParams
+        | name        | code   | reason                |
+        | <parameter> | <code> | <reason omschrijving> |
+
+        Voorbeelden:
+        | path      | request body                                                                                                                             | parameter            | code             | reason omschrijving                      |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((a b,c d))                                                                                       | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((134647.123))                                                                                    | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((134647.123 457842.456, 134747.789 d))                                                           | geometrie.intersects | geometryFormat   | Waarde is niet conform {format} formaat. |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON (([meer dan 350 coördinaten]))                                                                    | geometrie.intersects | maxItems         | Array bevat meer dan {maxItems} items.   |
+        | /adressen | AdresBody.geometrie.intersects=POLYGON ((5.96072575 52.18450437,5.95270058 52.17811002, 5.96793552 52.17816266, 5.96072575 52.18450437)) | geometrie.intersects | geometryMismatch | Waarde is niet conform opgegeven CRS.    |
+
+    Abstract Scenario: waarde van AdresBody.geometrie.intersects parameter overschrijdt maximale toegestane oppervlakte 
+        Als /adressen met request body AdresBody.geometrie.intersects=POLYGON ((134647 457842, 137512 457842, 137512 455907, 134647 455907, 134647 457842)) wordt aangeroepen
+        Dan bevat de response de volgende velden
+        | naam   | waarde                                        |
+        | title  | Een of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+        | code   | paramsValidation                              |
+        | detail | Bad request.                                  |
+        En bevat de response de volgende invalidParams
+        | name                           | code           | reason                                             |
+        | AdresBody.geometrie.intersects | surfaceMaximum | Waarde is hoger dan maximum oppervlakte 250000 m2. |
+
+Rule: verplichte request body is opgegeven
+
+    Scenario: verplichte request body is niet opgegeven
+        Als /adressen zonder request body AdresBody wordt aangeroepen
```

Comment:
hierbij moet je denk ik ook POST opnemen, anders kan je niet zien dat dit geen GET request is, die bijna (maar niet helemaal) dezelfde foutmelding geeft:
```
{
    "type": "https://tools.ietf.org/html/rfc7231#section-6.5.1",
    "title": "Ten minste één parameter moet worden opgegeven.",
    "status": 400,
    "detail": "Precies 1 parameter van pandIdentificatie, adresseerbaarObjectIdentificatie, zoekresultaatIdentificatie, postcode of huisnummer moet worden opgegeven",
    "instance": "https://api.bag.acceptatie.kadaster.nl/esd/huidigebevragingen/v1/adressen",
    "code": "paramsRequired"
}
```

Reply:
Als /adressen zonder request body AdresBody wordt aangeroepen

@MelvLee maakt dit het voor een developer (gebruiker) duidelijker om naast "request body" ook de naam van de schemacomponent (class) "AdresBody" te noemen? voor mij (niet-developer) klinkt dat dubbelop.

Reply:
Aangepast